client/snap_op.go

```diff
@@ -40,6 +40,28 @@ type SnapOptions struct {
 	IgnoreValidation bool   `json:"ignore-validation,omitempty"`
 }
 
+func (opts *SnapOptions) writeModeFields(mw *multipart.Writer) error {
+	for _, o := range []struct {
+		f string
+		b bool
+	}{
+		{"devmode", opts.DevMode},
+		{"classic", opts.Classic},
+		{"jailmode", opts.JailMode},
+		{"dangerous", opts.Dangerous},
+	} {
+		if !o.b {
+			continue
+		}
+		err := mw.WriteField(o.f, strconv.FormatBool(o.b))
```

Comment:
this can be just ... "true", no ?

(this makes me wonder if there's a hack based on the fact that this can be json marshalled that wouldn't be too dirty and would avoid some of this, also for action below)